### PR TITLE
Open Container Shell

### PR DESCRIPTION
**Pull request by jopit**
Signed-off-by: John Pitman <jspitman@ca.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [x] Enhancement

## What does this PR do ?

Adds an action to open a shell into the running application container

## Which issue(s) does this PR fix ?

https://github.com/eclipse/codewind/issues/2030

### CODE REVIEW

File: dev/src/main/java/org/eclipse/codewind/intellij/ui/actions/OpenContainerShellAction.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        return;
+                    }
+                    processArgs = "exec -n " + app.getNamespace() + " -it " + app.getPodName() + " -- " + command; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+                }
+                String cmd = processPath + " " + processArgs;
+                TerminalView terminalView = TerminalView.getInstance(myProject);
+                try {
+                    ShellTerminalWidget w = terminalView.createLocalShellWidget(null);
+                    w.executeCommand(cmd);
+                    w.requestFocusInWindow();
```
Comment (keithchong) [inline dev/src/main/java/org/eclipse/codewind/intellij/ui/actions/OpenContainerShellAction.java:71]: You know why the command is printed out in the console?  The docker command isn't supposed to be run inside the container.

**Comment by keithchong** [comment]
Hi @jopot, reviewed.  Just a couple of things to follow-up on.

**Comment by jopit** [comment]
The 'unhandled control sequence' messages show up in the log whenever you open an IntelliJ terminal, even if Codewind is not installed.  There's already an bug opened for it: https://youtrack.jetbrains.com/issue/IDEA-177768

File: dev/src/main/java/org/eclipse/codewind/intellij/ui/actions/OpenContainerShellAction.java (hunk 1 above)
Comment (jopit) [reply dev/src/main/java/org/eclipse/codewind/intellij/ui/actions/OpenContainerShellAction.java:71]: Eclipse doesn't show this, but VSCode does (see screen cap).  Eclipse terminal is different from either IntelliJ or VSCode.  I don't see an easy way to clear it, and I'm not sure it's worth pursuing.

image

Comment (keithchong) [reply dev/src/main/java/org/eclipse/codewind/intellij/ui/actions/OpenContainerShellAction.java:71]: Ok sounds good.

**Comment by keithchong** [comment]
> The 'unhandled control sequence' messages show up in the log whenever you open an IntelliJ terminal, even if Codewind is not installed. There's already an bug opened for it: https://youtrack.jetbrains.com/issue/IDEA-177768

Ok sounds good.

**Pull request merged**